### PR TITLE
Use env-less browserslist, autoprefix docs/style.css

### PR DESCRIPTION
**Pull request by jonaskuske**
This applies the changes I suggested in my 2.0 PR review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,11 +64,9 @@
     "stylelint": "^13.5.0",
     "stylelint-config-standard": "^20.0.0"
   },
-  "browserslist": {
-    "legacy": [
+  "browserslist":  [
       "defaults AND not android 4.4.3"
-    ]
-  },
+  ],
```
Comment (kognise) [inline package.json:69]: Is it worth tuning this config to make the bundle size smaller? Everything else looks great.

Comment (jonaskuske) [reply package.json:69]: `defaults` resolves to `> 0.5%, last 2 versions, Firefox ESR, not dead` which is pretty sensible I think.

These are the browsers that are included: https://browserl.ist/?q=defaults
But looks like "AND not android" part doesn't actually do anything [(Android Browser 76 is still included)](https://browserl.ist/?q=defaults+AND+not+android+4.4.3), so we can remove that.

Comment (jonaskuske) [reply package.json:69]: However updates to `defaults` could mean that our support matrix changes without us noticing, maybe explicitly listing everything we support is a better idea?

Comment (kognise) [reply package.json]: I don't think a changing support matrix matters that much, it might even be a good thing. Good point about removing android though.

Comment (jonaskuske) [reply package.json]: The only thing that'd matter is if they decided to remove IE11 from the defaults. But yeah that'd probably make the news on Twitter, Reddit etc. :D

So, :shipit: ?

Comment (kognise) [reply package.json]: :shipit:

**Pull request merged**